js/base/functions/type.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
     , safeFloat:          (o, k,          $default, n =   asFloat (prop (o, k))) => (isNumber (n)          ? n                         : $default)
     , safeInteger:        (o, k,          $default, n = asInteger (prop (o, k))) => (isNumber (n)          ? n                         : $default)
     , safeIntegerProduct: (o, k, $factor, $default, n = asInteger (prop (o, k))) => (isNumber (n)          ? parseInt (n) * $factor    : $default)
-    , safeTimestamp:      (o, k,          $default, n = asInteger (prop (o, k))) => (isNumber (n)          ? parseInt (n) * 1000       : $default)
+    , safeTimestamp:      (o, k,          $default, n = asInteger (prop (o, k))) => (isNumber (n)          ? parseInt (parseFloat (n) * 1000) : $default)
```

Comment:
@brandsimon is `parseFloat()` required here? Since `isNumber` is true, we could just multiply by a thousand and then `parseInt()`, what do you think?

Reply:
@kroitor 
Yes, I agree.